### DISCUSSION TITLE
Showcase

### DISCUSSION THREAD

**Discussion by crazko** [crazko/krystal#45] [Show and tell]
Do you use Krystal and have your TiddlyWiki published in public? Please, share the link here so we can see and learn from your work!

**Comment by arunnbabu81** [comment]
I am a big fan of Krystal. I use it in almost all of my wikis. 
This is how my wikis looks like - https://krystal.tiddlyhost.com/

**Comment by arunnbabu81** [comment]
Here is a version of Krystal with whitespace theme and sidebar

https://krytsal-whitespace.tiddlyhost.com
